test/built-ins/Object/subclass-object-arg.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2019 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-object-value
+author: Matthew Phillips <matthew@matthewphillips.info>
```

Comment:
@matthewp, I was not sure whether I could put your name to copyright, so I used `author` field.

Reply:
ianal, I give my permission to do what you need here though. I don't care for credit or anything. Thanks for raising this issue!